linear_elasticity/include/parameter_handling.h

```diff
@@ -265,6 +266,11 @@ namespace Linear_Elasticity
           "calculated-data",
           Patterns::Anything(),
           "Name of the write data in the precice-config.xml file");
+        prm.declare_entry(
+          "Flap location",
+          "0.0",
+          Patterns::Double(-3, 3),
```

Comment:
What does the `(-3, 3)` mean here?

(I also assume that `"0.0"` is the default value.

Reply:
These are upper and lower bounds for the specified values in the parameter file so that user can not exceed these positions.